### ISSUE TITLE
Device as argument not working for flows created in the Homey App (android)

### ISSUE THREAD

**Issue by StefanIndustries** [athombv/apps-sdk-issues#78]
Issue is related to #44 and #64.

Device as argument is passed on as Null when action card is created in the mobile app. When using the webbrowser flow editor everything is working fine.

Please see the 'error' reports below:
https://community.athom.com/t/bose-soundtouch/3775/28?u=slicinghaunt

https://community.athom.com/t/bose-sound-touch/10979/13?u=slicinghaunt
https://community.athom.com/t/bose-sound-touch/10979/14?u=slicinghaunt

**Comment by SlicingHaunt** [comment]
Is actually related to #64

**Comment by StefanIndustries** [comment]
```
        {
          "id": "create_zone_with",
          "title": {
            "en": "Create zone with",
            "nl": "Maak zone met",
            "de": "Zone erstellen mit"
          },
          "fluidArgs": true,
          "args":[
            {
              "name": "device",
              "type": "device",
              "filter": "driver_id=soundtouch"
            },
            {
              "name": "slave",
              "type": "device",
              "placeholder": {
                "en": "Soundtouch device",
                "nl": "Soundtouch apparaat",
                "de": "Soundtouch-Geräte"
              },
              "filter": "driver_id=soundtouch"
            }
          ]
        }
```

```
        const _createZoneWithAction = new Homey.FlowCardAction('create_zone_with')
            .register()
            .registerRunListener(async (args, state) => {
                //this.log('create zone', args);
                try {
                    console.log('creating zone started');
                    console.log(args.slave); // Null when created in the Homey app
                    console.log(args.device);// passes on the device, as it always did
                    await args.device._api.createZone(args.slave.getSettings().ip, args.slave.getSettings().mac);
                    return Promise.resolve(true);
                } catch (e) {
                    return promise.reject(e);
                }
            });
```

**Comment by Christian-Athom** [comment]
This issue is solved in Mobile App version 2.2.0
